es/lecciones/administracion-de-datos-en-r.md

```diff
@@ -0,0 +1,494 @@
+```
+
+Como ves, con tantos comandos anidados es difícil recordar cuántos paréntesis necesitas y hace que el código sea complicado de leer. Para mitigar esto algunas personas crean vectores temporales entre la llamada a cada función.
+
+```
+# Calcular la raíz cuadrada de la población de cada estado
+vector_raiz_cuadrada_poblacion_estados <- sqrt(importacion_poblacion_estados_eeuu$poblacion)
+
+# Calcular la suma de las raíces cuadradas de la variable temporal
+suma_del_vector_raices_cuadradas_poblacion_estados <- sum(vector_raiz_cuadrada_poblacion_estados)
+
+# Calcular la media de la variable temporal
+media_suma_del_vector_raices_cuadradas_poblacion_estados <- mean(suma_del_vector_raices_cuadradas_poblacion_estados)
+
+# Mostrar la mediana
+media_suma_del_vector_raices_cuadradas_poblacion_estados
+
+## [1] 1256925
+```
+
+Aunque vas a obtener la misma respuesta, esto es mucho más legible. Sin embargo, puede llenar tu espacio de trabajo con basura si olvidas borrar los vectores temporales. La línea de operaciones hace todo esto por ti. Aquí está el mismo código con la línea de operador _pipe_:
+
+```
+importacion_poblacion_estados_eeuu$poblacion%>%sqrt%>%sum%>%mean
+
+## [1] 1256925
+```
+Esto es mucho más fácil de leer e incluso lo podrías hacer más claro escribiéndolo en diferentes líneas
+
+```
+# Asegúrate de poner el operador al final de la línea
+importacion_poblacion_estados_eeuu$poblacion%>%
+  sqrt%>%
+  sum%>%
+  mean
+
+## [1] 1256925
+```
+
+Por favor, nota que los vectores o marcos de datos que la línea de operación crea son descartados cuando se completa la operación. Si quieres guardarlos, tienes que pasarlos a una nueva variable:
+
+```
+vector_raiz_y_suma_permanente_poblacion_estados <- importacion_poblacion_estados_eeuu$poblacion%>%sqrt%>%sum%>%mean
+vector_raiz_y_suma_permanente_poblacion_estados
+
+## [1] 1256925
+```
+
+## Necesitamos un nuevo conjunto de datos
+Ahora que entendemos la línea de operaciones, estamos preparados para empezar a mirar y administrar otro conjunto de datos. Desafortunadamente, para los historiadores, solo hay unos pocos conjuntos disponibles - ¡a lo mejor tú nos puedas ayudar a cambiar esto haciendo los tuyos públicos! Aquí contamos con el paquete [history data](https://cran.r-project.org/web/packages/historydata/index.html) (datos históricos) creado por [Lincoln Mullen](https://lincolnmullen.com).
+
+Vamos a instalar y cargar el paquete:
+
+```
+# Instalar el paquete historydata
+install.packages("historydata")
+
+# Cargar el paquete historydata
+library(historydata)
+```
+
+Este paquete contiene ejemplos de conjuntos de datos históricos - el ejemplo anterior con datos del censo del EEUU fue tomado de este paquete (y modificado por la traductora). A lo largo de este tutorial, vamos a trabajar con el conjunto de *early_colleges* (primeras_universidades) que contiene datos sobre las universidades fundadas antes de 1848.[^4] Lo primero que vamos a hacer es cargar los datos y leerlos:
```

Comment:
For text that references a function, file, or variable name like `early_colleges`, please use backticks like the other inline code segments.

Reply:
Done